### ISSUE TITLE
Case sensitve error with plugin detection

### ISSUE THREAD

**Issue by camlafit** [wpscanteam/wpscan#768]
Hi

wpscan looks problem with lower and upper case detection/result. In this case, plugin is updated but some older CVE are returned.

For example with wp-e-commerce, I've this result :
```
[+] Name: wp-e-commerce - v3.9.1
 |  Location: http://www.domain.tld/wp-content/plugins/wp-e-commerce/
 |  Readme: http://www.domain.tld/plugins/wp-e-commerce/readme.txt
[!] Directory listing is enabled: http://www.domain.tld/wp-content/plugins/wp-e-commerce/

[!] Title: WP-e-Commerce 3.8.9.5 - Cross Site Scripting Vulnerability
    Reference: https://wpvulndb.com/vulnerabilities/6555
    Reference: http://1337day.com/exploit/20517

[!] Title: WP-e-Commerce 3.8.9.5 - Remote Code Execution
    Reference: https://wpvulndb.com/vulnerabilities/6556
    Reference: http://packetstormsecurity.com/files/124921/
    Reference: http://osvdb.org/102484
    Reference: http://osvdb.org/102486

[!] Title: WP-e-Commerce 3.8.9.5 - Local File Inclusion
    Reference: https://wpvulndb.com/vulnerabilities/6557
    Reference: http://packetstormsecurity.com/files/124921/
    Reference: http://osvdb.org/102485

[!] Title: WP-e-Commerce 3.8.9.5 - GIF File Upload
    Reference: https://wpvulndb.com/vulnerabilities/6559
    Reference: http://packetstormsecurity.com/files/124921/
    Reference: http://osvdb.org/102497

[!] Title: WP e-Commerce <= 3.8.6 - SQL Injection Vulnerability
    Reference: https://wpvulndb.com/vulnerabilities/6563
    Reference: http://www.exploit-db.com/exploits/17832/
```

**Comment by erwanlr** [comment]
Hey there,

This has nothing to do with case sensitive things ;p, it's because the vulnerabilities don't have a fixed_in tag, i.e: we don't know which versions fix them

**Comment by ethicalhack3r** [comment]
I have contacted the vendor and asked for the fixed in information. Hopefully they will supply us with the information. If not, if anyone has the time and is willing, they can install the plugin versions associated with the vulnerabilities, attempt to exploit them and then see which other versions are also affected.

**Comment by camlafit** [comment]
Hi

Ok understood. 
Maybe add an information saiying "not fixed again" and "fixed since" then no risk to misunderstood wpscan result :)

**Comment by camlafit** [comment]
Hi

http://1337day.com/exploit/20517 looks solved at least in 3.9.1 I can't reproduce exploit in my website

**Comment by ethicalhack3r** [comment]
Thanks! Updated the issue - https://wpvulndb.com/vulnerabilities/6555 (will be auto exported tomorrow morning)

**Comment by erwanlr** [comment]
XSS fixed in 3.8.12: https://github.com/wp-e-commerce/WP-e-Commerce/commit/63955c3b725d6d2171da450f65cc7e3d9307b47a

I was unable to reproduce the RCE & LFI with the affected vulnerable version ... haven't fully checked the GIF issue but the vulnerable code seems to still be there. Not sure if those vulnd are real or just bullshit put along with a valid XSS to make them look legit :/.

SQLi fixed in 3.8.6.1

**Comment by ethicalhack3r** [comment]
Deleted

LFI: https://wpvulndb.com/vulnerabilities/6557
RCE: https://wpvulndb.com/vulnerabilities/6556
